src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/gitlab/GitlabMergeRequestDecorator.java

```diff
@@ -118,25 +116,18 @@ protected List<String> getCommitIdsForPullRequest(GitlabClient gitlabClient, Mer
     }
 
     @Override
-    protected void submitPipelineStatus(GitlabClient gitlabClient, MergeRequest mergeRequest, AnalysisDetails analysis, String sonarqubeRootUrl) {
+    protected void submitPipelineStatus(GitlabClient gitlabClient, MergeRequest mergeRequest, AnalysisDetails analysis,
+                                        AnalysisSummary analysisSummary) {
         Long pipelineId = analysis.getScannerProperty(PULLREQUEST_GITLAB_PIPELINE_ID)
                 .map(Long::parseLong)
                 .orElse(null);
 
-        BigDecimal coverage = analysis.getCoverage().orElse(null);
-
         try {
-            String dashboardUrl = String.format(
-                "%s/dashboard?id=%s&pullRequest=%s",
-                sonarqubeRootUrl,
-                URLEncoder.encode(analysis.getAnalysisProjectKey(), StandardCharsets.UTF_8.name()),
-                URLEncoder.encode(analysis.getBranchName(), StandardCharsets.UTF_8.name()));
-
             PipelineStatus pipelineStatus = new PipelineStatus("SonarQube",
                     "SonarQube Status",
                     analysis.getQualityGateStatus() == QualityGate.Status.OK ? PipelineStatus.State.SUCCESS : PipelineStatus.State.FAILED,
-                    dashboardUrl,
-                    coverage,
+                    analysisSummary.getDashboardUrl(),
+                    analysisSummary.getNewCoverage(),
```

Comment:
Hi, @mc1arke this change is regresion for https://github.com/mc1arke/sonarqube-community-branch-plugin/pull/225

Reply:
Hi @Karql! Rather than placing similar comments at different points in the project, could you raise an issue stating the problem and replication steps, and provide any links to changes you believe have caused the issue at that point? This way other people have visibility of the issue being reported, and can follow any discussion on it.

Reply:
Sorry I wanted to link these places together but forgot to past link to issue here.

Full details can be found here: https://github.com/mc1arke/sonarqube-community-branch-plugin/issues/285